packages/providers/src/ports/TokensApiProvider/Types/input.ts

```diff
@@ -0,0 +1,6 @@
+export interface ClaimCodeInput {
+  address: string;
+  qr_hash: string;
+  secret: string;
+  sendEmail: boolean;
+}
```

Comment:
I think we should not keep 1 on 1 relation between the legacy API and the SDK. We should name things based on our current name scheme and not keep eternally the legacy terms. /cc @alavarello

Reply:
:arrow_up: @nacho9900 @alavarello @rlajous